functions/tslint.json

```diff
@@ -0,0 +1,115 @@
+{
```

Comment:
we're using `eslint` throughout instead of `tslint` with a shared ruleset in the root of the repo.

Reply:
Also saw this one as well :)

The functions directory get compiled and managed by the Firebase CLI so I wanted to leave as “clean” as I could until we started using it.